go/vt/sqlparser/sql.y

```diff
@@ -319,6 +320,12 @@ select_statement:
     $$ = &Select{Comments: Comments($2), Cache: $3, SelectExprs: SelectExprs{Nextval{Expr: $5}}, From: TableExprs{&AliasedTableExpr{Expr: $7}}}
   }
 
+stream_statement:
+  STREAM comment_opt cache_opt distinct_opt straight_join_opt select_expression_list from_opt where_expression_opt group_by_opt having_opt
```

Comment:
Along the lines of the previous comment, let's simplify the syntax to allow only `STREAM comment_opt select_expression from_opt`. Note the singular select_expression.